READMEs/data-nfts-and-datatokens-flow.md

```diff
@@ -100,19 +100,12 @@ Congrats, you've created your first Ocean data NFT!
 In the same python console:
 ```python
 # Create ERC20 token related to the above NFT.
-from ocean_lib.web3_internal.constants import ZERO_ADDRESS
 
 nft_factory = ocean.get_nft_factory()
 erc20_token = erc721_nft.create_datatoken(
-    template_index=1, # default value
+    template_index=1,  # default value
     name="ERC20DT1",  # name for ERC20 token
     symbol="ERC20DT1Symbol",  # symbol for ERC20 token
-    minter=alice_wallet.address,  # minter address
-    fee_manager=alice_wallet.address,  # fee manager for this ERC20 token
-    publish_market_order_fee_address=alice_wallet.address,  # publishing Market Address
-    publish_market_order_fee_token=ZERO_ADDRESS,  # publishing Market Fee Token
-    publish_market_order_fee_amount=0,
-    bytess=[b""],
     from_wallet=alice_wallet
 )
```

Comment:
I would like to see this as a single line:
```python
erc20_token = erc721_nft.create_datatoken("datatoken 1", "DT1", from_wallet=alice_wallet)
```

Which after #832 will look like:
```python
datatoken = data_NFT.create_datatoken("datatoken 1", "DT1", from_wallet=alice_wallet)
```